src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/OgcMessageDistributor.java

```diff
@@ -0,0 +1,370 @@
+package de.terrestris.shogun2.util.interceptor;
+
+import java.text.MessageFormat;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import de.terrestris.shogun2.util.model.Response;
+
+/**
+ *
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+@Component
+public class OgcMessageDistributor {
+
+	/**
+	 * The Logger.
+	 */
+	private static final Logger LOG = Logger.getLogger(
+			OgcMessageDistributor.class);
+
+	/**
+	 *
+	 */
+	private static final String MODIFYING_REQUEST_MSG =
+			"Modifying a {0} {1} request";
+
+	/**
+	 *
+	 */
+	private static final String MODIFYING_RESPONSE_MSG =
+			"Modifying a {0} {1} response";
+
+	/**
+	 *
+	 */
+	private static final String REQUEST_IMPLEMENTATION_NOT_FOUND_MSG =
+			"No interceptor class implementation for request {0} {1} found. " +
+			"Forwarding the original request.";
+
+	/**
+	 *
+	 */
+	private static final String RESPONSE_IMPLEMENTATION_NOT_FOUND_MSG =
```

Comment:
currently unused, possibly a copy and past thing somewhere. Either change or remove this.